src/TestSuite/Migrator.php

```diff
@@ -112,7 +112,7 @@ public function runMany(
 
             if (!$migrations->migrate($migrationSet)) {
                 throw new RuntimeException(
-                    sprintf('Unable to migrate fixtures for `%s`.', $migrationSet['connection'])
+                    "Unable to migrate fixtures for `{$migrationSet['connection']}`."
```

Comment:
Personally, I like the sprintf() syntax more, in terms of readability etc. but no blocker.